investment-app/TRINV.Api/TRINV.Application/Queries/Dashboard/GetUserLatestInvestmentsQuery.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace TRINV.Application.Queries.Dashboard;
+
+using Domain.Entities;
+using Interfaces;
+using MediatR;
+using Shared.Business.Utilities;
+
+public record GetUserLatestInvestmentsQuery : IRequest<OperationResult<IList<GetUserLatestInvestmentsQueryModel>>>;
```

Comment:
You need to accepect Investment Type and use it inside currentUserInvestments, Currently you are returning all investment for this user, We need it based on the investment Type

Reply:
Done